src/grpc_client.cc

```diff
@@ -218,6 +235,8 @@ int32_t ClientCleanUpProc(grpc_labview::gRPCid* clientId)
         {
             (*activeClientCall)->Cancel();
         }
+
+        client->ActiveClientCalls.clear();
```

Comment:
Is `clear()` atomic? Should we get a lock here as well before we clear the list?

Reply:
yes, this is a scoped lock